TELCODOCS#2122: Configuring a local arbiter node (#92314)
openshift/openshift-docs: modules/nodes-cluster-enabling-features-about.adoc

** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
** Image mode behavior of image streams. Enables a new API for controlling the import mode behavior of image streams. (`imageStreamImportMode`)

> ** Configuring a local arbiter node. Enables users to configure an {product-title} cluster with two control plane nodes and one local arbiter node to retain high availability (HA) while reducing infrastructure costs. This configuration is supported only for bare-metal installations.

Reviewer: ```suggestion
** Configuring a local arbiter node. You can configure an {product-title} cluster with two control plane nodes and one local arbiter node so to retain high availability (HA) while reducing infrastructure costs for your cluster. This configuration is supported only for bare-metal installations.
```

![Screenshot From 2025-05-06 12-25-24](https://github.com/user-attachments/assets/61e18e89-f1a8-432d-b9f3-9fdbe6d7ebe1)
